.github/workflows/ci.yml

```diff
@@ -101,6 +105,23 @@ jobs:
               | tee out.txt; \
             echo ${PIPESTATUS[0]} > retcode.txt)
 
+      - name: Run CSV-only regression tests (no GAMS license)
+        env:
+          GAMS_LICENSE: ${{ secrets.GAMS_LICENSE }}
+        if: ${{ env.GAMS_LICENSE == '' }}
+        working-directory: xl2times
+        # Run without --dd flag GAMS license secret doesn't exist to see if we're just missing a GAMS license.
+        # This way we still run CSV-only regression tests without GAMS - useful for testing in forks before creating PRs.
+        run: |
+          source .venv/bin/activate
+          export PATH=$PATH:$GITHUB_WORKSPACE/GAMS/gams44.1_linux_x64_64_sfx
+          (python utils/run_benchmarks.py benchmarks.yml \
+              --times_dir $GITHUB_WORKSPACE/TIMES_model \
+              --verbose \
+              | tee out.txt; \
+          echo ${PIPESTATUS[0]} > retcode.txt)
+          echo 'Note: Pipeline will fail in final step due to missing GAMS license'
```

Comment:
Thanks.  You're right.  I decided to pass CI on forks with if the CSV regressions pass (because it'll still fail in in the main repo PR if the GAMS checks fail).  Just forgot to remove the echo here.
Fixed.